src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-remove-failed-node.ts

```diff
@@ -0,0 +1,240 @@
+    this._unsubscribe();
+  }
+
+  public async showDialog(
+    params: ZWaveJSRemoveFailedNodeDialogParams
+  ): Promise<void> {
+    this.entry_id = params.entry_id;
+    this.node_id = params.node_id;
+  }
+
+  public closeDialog(): void {
+    this._unsubscribe();
+    this.entry_id = undefined;
+    this._status = "";
+
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  public closeDialogFinished(): void {
+    history.back();
+    this.closeDialog();
+  }
+
+  protected render(): TemplateResult {
+    if (!this.entry_id || !this.node_id) {
+      return html``;
+    }
+
+    return html`
+      <ha-dialog
+        open
+        @closed="${this.closeDialog}"
+        .heading=${createCloseHeading(
+          this.hass,
+          this.hass.localize(
+            "ui.panel.config.zwave_js.remove_failed_node.title"
+          )
+        )}
+      >
+        ${this._status === ""
+          ? html`
+              <div class="flex-container">
+                <ha-svg-icon
+                  .path=${mdiRobotDead}
+                  class="introduction"
+                ></ha-svg-icon>
+                <div class="status">
+                  ${this.hass.localize(
+                    "ui.panel.config.zwave_js.remove_failed_node.introduction"
+                  )}
+                </div>
+              </div>
+              <mwc-button slot="primaryAction" @click=${this._startExclusion}>
+                ${this.hass.localize(
+                  "ui.panel.config.zwave_js.remove_failed_node.remove_device"
+                )}
+              </mwc-button>
+            `
+          : ``}
+        ${this._status === "started"
+          ? html`
+              <div class="flex-container">
+                <ha-circular-progress active></ha-circular-progress>
+                <div class="status">
+                  <p>
+                    <b>
+                      ${this.hass.localize(
+                        "ui.panel.config.zwave_js.remove_failed_node.in_progress"
+                      )}
+                    </b>
+                  </p>
+                </div>
+              </div>
+            `
+          : ``}
+        ${this._status === "failed"
+          ? html`
+              <div class="flex-container">
+                <ha-svg-icon
+                  .path=${mdiCloseCircle}
+                  class="error"
+                ></ha-svg-icon>
+                <div class="status">
+                  <p>
+                    ${this.hass.localize(
+                      "ui.panel.config.zwave_js.remove_failed_node.removal_failed"
+                    )}
+                  </p>
+                  ${this._error
+                    ? html` <p><em> ${this._error.message} </em></p> `
+                    : ``}
+                </div>
+              </div>
+              <mwc-button slot="primaryAction" @click=${this.closeDialog}>
+                ${this.hass.localize("ui.panel.config.zwave_js.common.close")}
```

Comment:
Btw, why doesnt this use `ui.common.close`?

Reply:
i tend to forget that generic translations like that exist 🙃